tests/e2e/globals.js

```diff
@@ -274,6 +283,11 @@ async function createAccountAndFundIt(browser, done, networkData) {
   await storeAccountData(browser, networkData)
   // switching to master account
   await switchToAccount(browser, networkData)
+  // funding main account
+  console.log(browser.globals.availableAtoms)

```

Comment:
remove? or if needed add some information what this value is

Reply:
Sorry, this needs to be removed